documentation/schema.json

```diff
@@ -1768,6 +1761,7 @@
           ]
         },
         "Filters": {
+          "description": "The filters that determine which exceptions should be included/excluded when collecting exceptions.",
```

Comment:
Any idea why this has changed?

Reply:
If you look at `OptionsDisplayStrings.resx`, you'll see that `DisplayAttributeDescription_CollectExceptionsOptions_Filters` was missing from the resx file but it exists in the backing cs file. This caused the string property to return `null` so the openapi generation would see a description value of `null`. This is probably the result of a bad merge.

When I removed `DisplayAttributeDescription_AzureAdOptions_SwaggerScope` from the resx file, it regenerated the backing cs file and removed the `DisplayAttributeDescription_CollectExceptionsOptions_Filters` property. So I added it back to the resx file with the expected value.

I can try to back out the changes around `DisplayAttributeDescription_CollectExceptionsOptions_Filters` and attempt to fix them separately.

Reply:
I'm fine with this as-is, it looks like our branches have this issue as well so we'd want both fixes backported anyways.